tests/functional/test_cache.py

```diff
@@ -124,11 +184,14 @@ def test_cache_dir_too_many_args(script, cache_dir):
     assert 'ERROR: Too many arguments' in result.stderr.splitlines()
 
 
-@pytest.mark.usefixtures("populate_wheel_cache")
-def test_cache_info(script, wheel_cache_dir, wheel_cache_files):
+@pytest.mark.usefixtures("populate_http_cache", "populate_wheel_cache")
+def test_cache_info(
+        script, http_cache_dir, wheel_cache_dir, wheel_cache_files
+):
     result = script.pip('cache', 'info')
 
-    assert 'Location: {}'.format(wheel_cache_dir) in result.stdout
+    assert 'HTTP files location: {}'.format(http_cache_dir) in result.stdout
```

Comment:
Thinking a bunch more, we should probably change this to day "Package index page cache location:" or something, instead of HTTP here.

Reply:
Updated!